### DISCUSSION TITLE
暂时无法访问面板数据接口..

### DISCUSSION THREAD

**Discussion by Msg-Lbo** [monsterxcn/nonebot-plugin-gspanel#37] [提问] [answered]
提示：暂时无法访问面板数据接口..
无其他报错

**Comment by monsterxcn** [comment]
仔细看一下 README

**Comment by monsterxcn** [comment]
![image](https://user-images.githubusercontent.com/22407052/192775391-3f5b0058-5162-4190-bb14-d539ff3d4873.png)

[Enka.Network](https://enka.network)

**Comment by ElainaFanBoy** [comment]
我也是之前用的好好的，今晚就开始这样了，无法访问面板端口

**Comment by monsterxcn** [comment, marked as answer]
恼，怎么都不看我写的说明吗？上面那么大个图也看不见吗？

再 Copy 一下 README 使用须知部分：

> 插件返回「暂时无法访问面板数据接口..」可能的原因有：Bot 与 [Enka.Network](https://enka.shinshin.moe/) 的连接不稳定；[Enka.Network](https://enka.shinshin.moe/) 服务器暂时故障等。

现在这种情况也算 **[Enka.Network](https://enka.shinshin.moe/) 服务器暂时故障** 这类，上游会在每个大版本更新的时候进入维护状态，你自己去查一下就能看到我上面那个图片一样的结果，解决办法就是等上游维护完。你用其他查面板的工具基本也是一样的结果，不要反复问了，等一阵子就好了。

**Comment by monsterxcn** [comment]
另外新角色的面板也要靠上游的几个仓库更新之后我这边才好作支持。

其中包括：

1. [@Dimbreath/GenshinData](https://github.com/Dimbreath/GenshinData)：从该仓库的文件生成完整翻译资源
2. [@yoimiya-kokomi/miao-plugin](https://github.com/yoimiya-kokomi/miao-plugin)：从该仓库的文件 [/resources/meta/artifact/artis-mark.js](https://github.com/yoimiya-kokomi/miao-plugin/blob/master/resources/meta/artifact/artis-mark.js) 获取角色圣遗物评分计算规则

**Comment by JY69** [comment]
请问可以自定义更换面板查询接口么，b服的经常访问不了

**Reply by monsterxcn** [reply]
你是想用 miao-api 吗？

**Reply by JY69** [reply]
> 你是想用 miao-api 吗？

用miao 或者MiniGrayGay大佬的这两条，b服实在太难了。
https://enka.minigg.cn/
https://enka.microgg.cn/

**Reply by monsterxcn** [reply]
这个已经内置了，并且失败会自动重试

**Reply by monsterxcn** [reply]
小灰灰的这两个

**Reply by monsterxcn** [reply]
明白了，Enka 的响应有变化，导致 B 服并没有从小灰灰的 API 重试。近期我会修改一下，抱歉！

**Reply by JY69** [reply]
> 明白了，Enka 的响应有变化，导致 B 服并没有从小灰灰的 API 重试。近期我会修改一下，抱歉！

好的，谢谢大佬。这边b服的群友可期待用，那个组队伤害功能。

**Reply by JY69** [reply]
> 明白了，Enka 的响应有变化，导致 B 服并没有从小灰灰的 API 重试。近期我会修改一下，抱歉！

还有请问下，大佬处理后，这边需要操作什么更新么？还是直接使用即可？

**Reply by monsterxcn** [reply]
> 还有请问下，大佬处理后，这边需要操作什么更新么？还是直接使用即可？

还没发版，等我把 3.5 的东西更新完之后再发新版，到时候更新插件重启
